### PR TITLE
Disable registration efforts on JeOS and CaaSP

### PR DESCRIPTION
**Pull request by mnowaksuse**
http://assam.suse.cz/tests/5623#step/bootloader_uefi/3

### CODE REVIEW

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -98,17 +98,22 @@ sub bootmenu_default_params {
...
-        type_string_very_slow "plymouth.ignore-serial-consoles ";    # make plymouth go graphical
-        type_string_very_slow "linuxrc.log=$serialdev ";             # to get linuxrc logs in serial
-        type_string_very_slow "console=$serialdev ";                 # to get crash dumps as text
-        type_string_very_slow "console=tty ";                        # to get crash dumps as text
-
-        assert_screen "inst-consolesettingstyped", 30;
+    if (!get_var("NICEVIDEO")) {
+        if (is_jeos) {
+            type_string_very_slow "console=$serialdev ";    # to get crash dumps as text
+            type_string_very_slow "console=tty ";           # to get crash dumps as text
```
Comment (okurz) [inline lib/bootloader_setup.pm]: why duplicate these lines?

File: tests/installation/bootloader_uefi.pm [hunk 2]
```diff
@@ -120,7 +120,7 @@ sub run() {
     bootmenu_default_params;
     specific_bootmenu_params;
 
-    registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED);
+    registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED) unless is_jeos;
```
Comment (okurz) [inline tests/installation/bootloader_uefi.pm]: can you state why?

Comment (mnowaksuse) [inline tests/installation/bootloader_uefi.pm]: In JeOS we don't have linuxrc present, hence the code is not applicable.

Comment (okurz) [inline tests/installation/bootloader_uefi.pm]: … in code

**Comment by mnowaksuse** [comment]
As there are no objections (and review are not coming), I intend to merge it tomorrow.

**Pull request merged**